dev/reference/sql/constraints.md

```diff
@@ -12,7 +12,7 @@ TiDB supports the same basic constraints supported in MySQL with the following e
 
 - `PRIMARY KEY` and `UNIQUE` constraints are checked lazily by default. By batching checks until when the transaction commits, TiDB is able to reduce network communication. This behavior can be changed by setting `tidb_constraint_check_in_place` to `TRUE`.
 
-- `FOREIGN KEY` constraints are not currently enforced by DML.
+- TiDB supports creating foreign key constraints, but these constraints are ineffective in DML statements.
```

Comment:
```suggestion
- TiDB supports creating foreign key constraints, but these constraints are not enforced by DML statements.
```
@wjhuang2016 Please take a look at this update suggested by @gregwebs .

Reply:
LGTM